global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "6.0.100-rc.1.21463.6"
```

Comment:
This is a tricky one. Do you really want anyone contributing to the entire repo to require rc1?

Reply:
At some point, this is going to be required for changes like https://github.com/domaindrivendev/Swashbuckle.AspNetCore/pull/2210. @domaindrivendev mentioned being OK with upgrading to a publicly available RC is fine. I think since we say our RCs are go-live, this should be fine.

The alternative is to remove the tests for this but I think we're gonna benefit greatly from having a test for top-level apps in the repo.